bin/runtime-common/src/messages_call_ext.rs

```diff
@@ -20,10 +20,10 @@ use crate::messages::{
 use bp_messages::{LaneId, MessageNonce};
 use frame_support::{dispatch::CallableCallFor, traits::IsSubType};
 use pallet_bridge_messages::{Config, Pallet};
-use sp_runtime::{transaction_validity::TransactionValidity, RuntimeDebug};
+use sp_runtime::transaction_validity::TransactionValidity;
 
 /// Info about a `ReceiveMessagesProof` call which tries to update a single lane.
-#[derive(Copy, Clone, PartialEq, RuntimeDebug)]
+#[derive(PartialEq, Debug)]
```

Comment:
IIRC using `Debug` (and all string operations) wasn't desired in runtime. So maybe revert to `RuntimeDebug` here?

Reply:
You're right. I forgot. Done.